<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgLayerSetDialog</class>
 <widget class="QDialog" name="ImgLayerSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Layer Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>415</width>
        <height>800</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Start Jet Offset:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="startJetOffLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Print Direction:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="prtDirComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>X Offset (mm):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="xPosOffLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Y Offset (mm):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="prtYOffetLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color Count:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="colorCntsLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>X DPI:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="xDPILineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Y DPI:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="yDPILineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Start Pass Index:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="startPassIndexLineEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Feather Mode:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="featherModeComboBox"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Custom Feather Jets:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="customFeatherJetsLineEdit"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Edge Scale Pixel:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="edgeScalePixelLineEdit"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Rotate Angle:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="rotateAngleLineEdit"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Destination X Scale:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="dstXScaleLineEdit"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Destination Y Scale:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="dstYScaleLineEdit"/>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Layer Density:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLineEdit" name="layerDensityLineEdit"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Edge Density:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLineEdit" name="edgeDensityLineEdit"/>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>X Scan Speed:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QLineEdit" name="xScanSpdLineEdit"/>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Image Type:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QComboBox" name="imgTypeComboBox"/>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Scan Control:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QComboBox" name="scanCtlValueComboBox"/>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Ink Multiplication:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QComboBox" name="mulityInkComboBox"/>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Pass Volume:</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="QComboBox" name="passVolComboBox"/>
       </item>
       <item row="21" column="0" colspan="2">
        <widget class="QGroupBox" name="flagsGroupBox">
         <property name="title">
          <string>Flags</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="doubleDirCheckBox">
            <property name="text">
             <string>Bi-Directional Print</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="yReverseCheckBox">
            <property name="text">
             <string>Y Reverse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="xInsertCheckBox">
            <property name="text">
             <string>X Insert</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="switchByWhiteCheckBox">
            <property name="text">
             <string>Switch by White</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="edgeNoReduceCheckBox">
            <property name="text">
             <string>Edge No Reduce</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="prtAreaCheckBox">
            <property name="text">
             <string>Prt Area</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="grayBitsCheckBox">
            <property name="text">
             <string>2-bit Grayscale</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImgLayerSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImgLayerSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
